### ISSUE TITLE
Questions about quick start

### ISSUE THREAD

**Issue by dgleba** [ais-one/vue-crud-x#76]
Thanks for this project.  I like it. Thanks for your work.

I did this.

```
  # Settings.
    dir=/srv/test
    proj=vuecrudx

  mkdir -p $dir
  cd $dir
  git clone https://github.com/ais-one/vue-crud-x.git $proj
  cd $proj
  cd example-spa
  
  # -------  edit .env  ---------
  cp .env-sample .env
  # -------  edit .env  ---------

  # -------  edit .env  ---------
  cp ../backend/.env-sample ../backend/.env
  # -------  edit .env  ---------
  
  npm i
  npm run build-rest
  npm run init-db
  npm run start
```

It seems to work for many things. I can login and create a book, etc.
I get errors in the console. See pics.

_____________

![2019-10-20 17_59_42-VueCrudX Example](https://user-images.githubusercontent.com/4622715/67168317-85040d00-f371-11e9-9642-9d4bb55965cf.jpg)

_____________

![2019-10-20 19_03_30-VueCrudX Example](https://user-images.githubusercontent.com/4622715/67168316-846b7680-f371-11e9-8ef9-de20ebb1a07c.jpg)

_____________

Are the errors normal?
Did I do something wrong?

Why does it show the  "find error. network error. failed to fetch" after about 5 minutes?

**Comment by dgleba** [comment]
After seeing the note in the `.env-sample` file, I changed it to the following..
```
  cp .env-sample .env.development
  cp ../backend/.env-sample ../backend/.env.development
```
I edited the env files as follows

Front..
```
VUE_APP_API_URL=http://192.168.88.39:3000
# VUE_APP_WS_URL=ws://192.168.88.39:3001
VUE_APP_WS_URL=
```
Back..
```
API_PORT=3000
# WS_PORT=3001
WS_PORT=
```

I still get the same errors.

**Comment by ais-one** [comment]
Hi,

If you are on local machine

Frontend:
VUE_APP_API_URL=http://127.0.0.1:3000
VUE_APP_WS_URL=ws://127.0.0.1:3001

Backend:
API_PORT=3000
WS_PORT=3001

If not replace the 127.0.0.1 with your IP. check that the ports are not blocked...

**Comment by ais-one** [comment]
Hi,

I found a bug in the backend regarding websockets. am fixing it and it will be in the next release

**Comment by ais-one** [comment]
Resolved in 0.2.5

https://github.com/ais-one/vue-crud-x/pull/77
